section1/reflection.md

```diff
@@ -1,19 +1,33 @@
 ## Section 1 Reflection
 
 1. Regarding the blog posts in Part A, how do you feel about asking questions? Do you tend to ask them too soon, or wait too long, or somewhere in between?
+* I am usually a little nervous about asking questions but I will still ask them. I tend to be somewhere in the middle.
 
 2. What are the data types you learned about in this section? In your own words, define each.
+* String: It is usually words or characters that are used inside a quote. It will read exactly whats in the quotes
+* Number: Used for all numbers
+* Boolean: a statement that has a value of either true or false
 
 3. How would you log the string `"Hello World!"` to the console?
+`console.log('Hello World!')`
 
 4. What is/are the character(s) you would use to indicate comments in a JavaScript file? What is the purpose of a code comment?
+* You use two backslashes => //
+* the purpose is to comment on what you are doing. you may say what this specific line of code is doing, or use it for a note to yourself later down the line.
 
 5. In your own words, what is a variable? How would you explain it to a 5 year old?
+* A variable is like a label for an item. You can think of a variable like a name tag. I take the name tag. i write what I want on it and then when I place it on an object you could now use the term on the name tag to describe the object.
 
 6. Think of a site or app you use frequently. What are three variables that are probably used? Which data type would each of those variables probably hold?
+* Username/password/email
+* Username would probably hold a string.
+* Password would probably hold a string.
+* Email would hold a string.
 
 7. In your own words, explain what concatenation is.
+* it is when you join things together. It is almost like a small addition problem but not only adding numbers you can add variables, strings, numbers, etc together to form one piece of information.
 
 8. Think of a site or app you use frequently. Where do you think the developers used concatenation?
+* Probably on the greeting message when you first log in. It will have a general message laid out but put in personal info such as your name.
 
 9. What questions do you still have about the work we've done so far? (not having a question is not an option)
```

Comment:
Make sure you're reading the directions and questions. I won't make you think back to questions from section1 now...but that you skipped this is concerning.